server/service/src/invoice/outbound_shipment/update/mod.rs

```diff
@@ -417,6 +426,7 @@ mod test {
                 their_reference: Some("their_reference".to_string()),
                 colour: Some("colour".to_string()),
                 transport_reference: Some("transport_reference".to_string()),
+                tax: None,
```

Comment:
From the comment: "// Test all fields apart from status" I think tax should be set to some...

Reply:
[cb099e7](https://github.com/openmsupply/open-msupply/pull/919/commits/cb099e728e50ec5073250d4cbd8bf1c82cefdb91)